client/oci_env/utils.py

```diff
@@ -248,14 +267,12 @@ def exec(self, args, service=None, interactive=False, pipe_output=False):
         project_name = self.config["COMPOSE_PROJECT_NAME"]
         binary = self.config["COMPOSE_BINARY"]
 
-        container = f"{project_name}_{service}_1"
-
         # docker fails on systems with no interactive CLI. This tells docker
         # to use a pseudo terminal when no CLI is available.
         if os.getenv("COMPOSE_INTERACTIVE_NO_CLI", "0") == "1":
-            cmd = [binary, "exec", container] + args
+            cmd = [binary, "exec", self.container_name] + args
```

Comment:
This is designed to work with multiple services (not just pulp) so that you can configure the system to run DBs, workers and content app services in other containers as well.

Can you update `container_name` to take in the `service` arg instead of hard coding this to be `pulp`?